src/app/_components/TaskList.tsx

```diff
@@ -0,0 +1,99 @@
+import { useState } from "react";
+
+interface checkListItem {
+  id: number;
+  task: string;
+  isChecked: boolean;
+}
+
+export default function TaskList() {
+  const [taskList, setTaskList] = useState<checkListItem[]>([]);
+  const [task, setNewTask] = useState<string>("");
+  const [incrementID, setIncrementID] = useState<number>(0);
+
+  const addedTasks = (text: string) => {
+    const newTaskList: checkListItem = {
+      id: incrementID,
+      task: text,
+      isChecked: false,
+    };
+
+    setTaskList([...taskList, newTaskList]);
+    setNewTask("");
+    setIncrementID(incrementID + 1);
+  };
+
```

Comment:
Instead of incrementing the id number by one, can you use some other method of handling duplicates. (HINT: Sets)